src/main/java/com/scalar/db/benchmarks/tpcc/TpccUtil.java

```diff
@@ -89,14 +89,12 @@ public static int getItemId() {
    */
   public static String getRandomStringWithOriginal(int minLength, int maxLength, int rate) {
     int length = randomInt(minLength, maxLength);
+    StringBuilder builder = new StringBuilder(TpccUtil.randomAlphaString(length));
     if (TpccUtil.randomInt(0, 99) < rate) {
-      int startOriginal = TpccUtil.randomInt(2, length - 8);
-      return TpccUtil.randomAlphaString(startOriginal - 1)
-          + "ORIGINAL"
-          + TpccUtil.randomAlphaString(length - startOriginal - 9);
```

Comment:
Min and max values in `randomInt()` are both **inclusive**, so there was a chance to call randomAlphaString() with -1, which causes `NegativeArraySizeException`.

A detailed background is as follows. Originally, `randomInt()` used a bit different method to calculate a random number. But it had a bug to return a value greater than max (e.g., randomInt(-1, -1) returns 0) due to the cast and round-off fractions. So the issue did not appear, fortunately, until now. Now we changed it like the following and correctly returns -1, that's why the issue appeared. Both issues were brought from BenchBase which I referred to, though...

https://github.com/scalar-labs/scalardb-benchmarks/blob/f4a93c58178219424e2f53679409a4d43a3ea9fc/src/main/java/com/scalar/db/benchmarks/tpcc/TpccUtil.java#L157-L159